### DISCUSSION TITLE
过期：input.conf里的"MBTN_LEFT  ignore" 无视鼠标左键点击有意义吗

### DISCUSSION THREAD

**Discussion by yeezylife** [hooke007/mpv_PlayKit#74] [回收]
mpv.conf里有 `no-input-default-bindings`

然后MBTN_LEFT相关的东西什么都不设置,点左键本来也没有任何操作啊
需要ignore吗? 有点疑惑...

**Comment by hooke007** [comment]
没有，只是忘了他的存在（

**Reply by yeezylife** [reply]
当前input.conf里面有多行相关的 `ignore` !!!

**Reply by hooke007** [reply]
no-input-default-bindings
引入后忘了处理那些空键了，实际也没啥影响就一直搁置了
